### PR TITLE
Migrate timezone and calendar icon

### PR DESCRIPTION
**Pull request by GretaD**
ref https://github.com/nextcloud/groupware/issues/38

### CODE REVIEW

File: src/components/Shared/DatePicker.vue [hunk 1]
```diff
@@ -46,10 +46,13 @@
 				@click.stop.prevent="toggleTimezonePopover"
 				@mousedown.stop.prevent="() => {}">
 				<template #icon>
-					<IconTimezone v-if="!isAllDay"
-						:size="20" />
 					<IconNewCalendar v-if="isAllDay"
 						:size="20" />
+					<IconTimezone v-if="!isAllDay && highlightTimezone"
```
Comment (tcitworld) [inline src/components/Shared/DatePicker.vue:51]:
```suggestion
					<IconTimezone v-else-if="highlightTimezone"
```

File: src/components/Shared/DatePicker.vue [hunk 2]
```diff
@@ -46,10 +46,13 @@
...
+						class="highlighted-timezone-icon"
+						:size="20" />
+					<IconTimezone v-if="!isAllDay && !highlightTimezone"
```
Comment (tcitworld) [inline src/components/Shared/DatePicker.vue:54]:
```suggestion
					<IconTimezone v-else-if="!highlightTimezone"
```
Or just `v-else`

File: src/components/Shared/DatePicker.vue [hunk 3]
```diff
@@ -36,16 +36,25 @@
...
-				@mousedown.stop.prevent="() => {}" />
+				@mousedown.stop.prevent="() => {}">
+				<template #icon>
+					<IconNewCalendar v-if="isAllDay"
+						:size="20" />
+					<IconTimezone v-else-if="highlightTimezone"
+						class="highlighted-timezone-icon"
+						:size="20" />
+					<IconTimezone v-else-if="!highlightTimezone"
+						:size="20" />
```
Comment (tcitworld) [inline src/components/Shared/DatePicker.vue:55]:
```suggestion
					<IconTimezone v-else
						:class="{ 'highlighted-timezone-icon': highlightTimezone }"
						:size="20" />
```
Sorry, thought of it afterwards.

Comment (GretaD) [reply src/components/Shared/DatePicker.vue:55]: ok, i will change it, but it does work like it is though

Comment (tcitworld) [reply src/components/Shared/DatePicker.vue]: Yes, the class will be applied only if `highlightTimezone` is `true`.

Comment (GretaD) [reply src/components/Shared/DatePicker.vue]: done

**Review by miaulalala — APPROVED** [review]

**Pull request merged**